moderatorFrontend/src/main/kotlin/views/accessManagement/AccessManagementDetails.kt

```diff
@@ -257,6 +250,9 @@ class AddLocation(props: AccessManagementDetailsProps) : RComponent<AccessManage
   }
 
   private fun RBuilder.renderTimeSlotPickers() {
+    val now = Date()
+    val DAYS_PER_YEAR = 365
+    val inThreeYears = now.addDays(DAYS_PER_YEAR * 3)
```

Comment:
Should we add an `addYears()` function to our Date extensions? I guess we should use setFullYear() under the hood.  https://stackoverflow.com/a/33924045

Also backport to webcore if you decide to do it :)

Reply:
I like this suggestion 👍